app/utilities/schema.py

```diff
@@ -96,17 +98,18 @@ def _schema_exists(language_code, schema_name):
     )
 
 
-def get_allowed_languages(schema_name, launch_language):
-    for language_combination in LANGUAGES_MAP.get(schema_name, []):
-        if launch_language in language_combination:
-            return language_combination
+def get_allowed_languages(schema_name: str | None, launch_language: str):
+    if schema_name:
```

Comment:
This shouldn't ever be called without a schema_name?. The metadata would. be mandatory.

Reply:
![Screenshot 2023-04-24 at 09 28 43](https://user-images.githubusercontent.com/108286538/233941672-88b93b71-7522-4ee0-9065-c24b422b2eea.png)

Session store/data existing ensures metadata is not None and also that schema name is not None yeah? If so will change method to `get_allowed_languages(schema_name: str, launch_language: str)` and `type ignore` 👍

Reply:
I would guard against schema name not existing, it will always exist at the moment but near future that would be the case. So i would update `handle_language` to ensure you have schema name.

Reply:
Discussed via call, no action needed as usage of schema_name will be changed in future as we integrate with CIR